README.md

```diff
@@ -1,18 +0,0 @@
-# doc-resources
-Ontotext documentation resources
-
-Adding this repository as submodule to another repository
```

Comment:
I don't remember. I think there were some problems with the my local versioning. Restore it. Sorry.